docs: add docs for CST commands (#8090)
confluentinc/ksql: docs/developer-guide/ksqldb-reference/create-table.md

If the IF NOT EXISTS clause is present, the statement won't fail if a
table with the same name already exists.

### Source tables

> If the SOURCE clause is present, then pull queries can be executed on it. Note that the SOURCE

Reviewer: ```suggestion
If the SOURCE clause is present, you can execute pull queries on it. The SOURCE

```

> clause will execute an internal query for this table in order to create a materialized state which

Reviewer: ```suggestion
clause executes an internal query for the table to create a materialized state that's

```

> is used by pull queries. This query cannot be terminated manually. This query is only terminated

Reviewer: ```suggestion
used by pull queries. You can't terminate this query manually. You can terminate it only

```

> when dropping the table with DROP TABLE.

Reviewer: ```suggestion
by dropping the table with the DROP TABLE statement.

```

> Also, the table will be created as read-only when the SOURCE clause is provided. INSERTS statements

Reviewer: ```suggestion
Also, when you provide the SOURCE clause, the table is created as read-only. For a read-only table, INSERT statements

```

> and the DELETE TOPIC on DROP TABLE statements won't be permitted.

Reviewer: ```suggestion
and the DELETE TOPIC on DROP TABLE statements aren't permitted.

```

> To disable the SOURCE table feature, you can set `ksql.source.table.materialization.enabled` to

Reviewer: ```suggestion
To disable the SOURCE table feature, set `ksql.source.table.materialization.enabled` to

```

> `false` in your KSQL server properties file.

Reviewer: ```suggestion
`false` in your ksqlDB server properties file.

```
